flake.lock

```diff
@@ -83,11 +83,11 @@
         ]
       },
       "locked": {
-        "lastModified": 1718003967,
-        "narHash": "sha256-PJbw9L8NgRKYkKqc1Pywv1glSyT+iE1D79GuF7ec4tc=",
+        "lastModified": 1725522308,
+        "narHash": "sha256-AtsFZ92WkVkH8fd0Xa0D6/PR84/dtOH6gpM4mtn32Hk=",
         "owner": "edgelesssys",
         "repo": "uplosi",
-        "rev": "b7edeef9b9eb1226998c86ffc8fbf9eacce94549",
+        "rev": "c8a482de299b5442f558d7381f0cbf898e0de0e1",
```

Comment:
The commit description says we're upgrading mkosi, but the diff looks like we're upgrading only uplosi. Is the upgrade transitive, or am I missing something?

Reply:
Yeah - This is a little confusing. The bump to `nixpkgsUnstable` above actuall brings in the [commit that updated mkosi](https://github.com/NixOS/nixpkgs/pull/338962) in nixpkgs.